core/chains/chain.go

```diff
@@ -38,6 +38,7 @@ const (
 	SOL   Token = "SOL"
 	STARS Token = "STARS"
 	XNO   Token = "XNO"
+	SUI   Token = "SUI"
```

Comment:
Please place `SUI` below `STARS`
```suggestion
	STARS Token = "STARS"
	SUI   Token = "SUI"
	XNO   Token = "XNO"
```